### PR TITLE
MCP-2297 v2 PDF upload test

### PR DESCRIPTION
**Pull request by ianwolson**
## What was the problem?
We need to test the evidence-pdf endpoint for both its generation and fetching.

Associated tickets or Slack threads:

- [MCP-2297](https://amida.atlassian.net/browse/MCP-2297)

## How does this fix it?[^1]
Test which runs a pdf generation and test whether we can fetch the pdf.

## How to test this PR
- Run Vro v2 end2end tests

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

File: app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java [hunk 1]
```diff
@@ -116,6 +124,19 @@ void testAutomatedClaim_missingAnchor() {
...
+  void testPdfGenerationAndFetch() throws Exception {
+    TestSetup setup = TestSetup.getInstance("test-7101-01");
+    helper.setApiKey("test-key-01");
+    ResponseEntity<String> response = helper.generatePdf(setup);
+    Assertions.assertEquals(HttpStatus.OK, response.getStatusCode());
+    String actual = response.getBody();
+    String expected = setup.getGeneratePdfResponse();
+    JSONAssert.assertEquals(expected, actual, JSONCompareMode.STRICT);
+    ResponseEntity<byte[]> fetchResponse = helper.getPdf(setup);
+    Assertions.assertEquals(HttpStatus.OK, fetchResponse.getStatusCode());
```
Comment (jbregstone) [inline app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java:137]: I was imagining this assertion as part of the `testAutomatedClaim` test. That endpoint should kick off the process that leads to generating a pdf. For end2end testing we want to make sure that it actually gets to that point in the  process.

Comment (jbregstone) [reply app/src/end2endTest/java/gov/va/vro/end2end/VroV2Tests.java:137]: You may need to add in an await function to give it a minute for the processing to go through

**Comment by jbregstone** [comment]
This PR may no longer be necessary because of https://github.com/department-of-veterans-affairs/abd-vro/pull/1045

**Comment by ianwolson** [comment]
Closing this PR, #1045 covers what this test was supposed to do.

**Pull request closed without merging**